core/src/smartcontracts/isi/asset.rs

```diff
@@ -150,6 +150,7 @@ pub mod isi {
     impl_transfer!(Fixed, "transfer_fixed");
 
     /// Trait for blanket mint implementation.
+    #[allow(clippy::trait_duplication_in_bounds)] // False positive
```

Comment:
Maybe file an issue to clippy? They had something [similar](https://github.com/rust-lang/rust-clippy/issues/9076) fixed recently with this lint, though.

Reply:
I checked this bug for rust 1.63-65, error message is gone on rust 1.65.